tests/datasource/fluent/integration/integration_test_utils.py

```diff
@@ -32,131 +26,6 @@
 logger = logging.getLogger(__name__)
 
 
-def run_checkpoint_and_data_doc(
```

Comment:
I think this function is used by a lot of fluent tests to verify checkpoints are running successfully. I'm surprised it still works because it uses the old validator workflow, but maybe we haven't ripped that out yet.

This tests a lot more than saving validations. For example, it tests datadocs are being generated.

I'm wondering if we can do something more precise than completely deleting this.

Reply:
Hmm so due to the fact that I've removed actions and therefore removed the ability to persist validation results with v0.18 checkpoints, data docs don't really work within our test suite. Things are in a bit of a weird intermediate state I do agree that we should probably refrain from a large deletion like this.